javascript.js

```diff
@@ -78,3 +78,25 @@ function createElement(htmlString) {
 
```

Comment:
- Kindly enable your ``ESlinters`` in your pull request.

**_You can have setup files [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)._**